Sources/Plasma/PubUtilLib/plAvatar/plAvBrainCoop.cpp

```diff
@@ -151,7 +151,7 @@ bool plAvBrainCoop::RelayNotifyMsg(plNotifyMsg *msg)
 
     proMultiStageEventData * mtevt = static_cast<proMultiStageEventData *>(msg->FindEventRecord(proEventData::kMultiStage));
     if(mtevt)
-        DebugMsg("COOP: Relaying multi-stage event to %d recipients (via plAvBrainCoop)", fRecipients.size());
+        DebugMsg("COOP: Relaying multi-stage event to %lu recipients (via plAvBrainCoop)", fRecipients.size());
```

Comment:
```suggestion
        DebugMsg("COOP: Relaying multi-stage event to %zu recipients (via plAvBrainCoop)", fRecipients.size());
```

(`z` is the length modifier for `size_t`)